src/assets/styles/theme.scss

```diff
@@ -27,6 +27,7 @@ $BCgovABlue1: #6e93ff;
 $BCgovABlue2: #3b6cff;
 $BCgovABlue3: #0846ff;
 $BCgovABlue4: #003bee;
+$BCgovBlue5O: #01336626; // LOW OPACITY
```

Comment:
I thought hex colors had to be 6 (or 3) chars. Does this work?

Reply:
Indeed it does!
On the web, you can add the transparency code to the end of the hex, so in this case `26` will apply a 15% transparency, i think for mobile the code goes at the start?
Forgive me, i can't recall the exact origination of the colour but i imagine it was pulled directly from some previous design specs for assets and copied over to the uxpin for the credentials pilot.

Reply:
https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4

Reply:
That gist shows the transparency code at the front...?

Reply:
Suggestion: update comment to indicate level of opacity or format (for when I look at this again and have forgotten everything).

I leave it in your capable hands :)

Reply:
haha totally, will do!